### PR TITLE
Add some options to control "chattiness" of the plugin

### LINKED ISSUE #34: [FEAT] Make skill less chatty
### Objective

Some actions performed by this plugin have obvious visual feedback - turning off lights, for instance. I would like to have an option to disable mycroft voice response on those actions. Something like blacklist with entities names. If you like this feature, I can try to implement it and create PR

### Initial Implementation Requirements

-

### Other Considerations

_No response_

### PR DESCRIPTION
**Pull request by jumper047**
I added two options to your plugin:
- Skip `acknowledge` dialogue completely (I believe it is have not too much of useful information, because just after it another dialog started which actually duplicates information from first one. In fact, I'd suggest to get rid of it completely). Option disabled by default.
- Skip confirmation dialogue in actions for certain entities. Usecase: control lights in the same room as Mycroft. I don't need additional confirmation that lights are turned on if they are turned on.

Settings file example with new params:
```
{
    "verbose": false,
    "silent_entities": [
        "hallway lamp"
    ],
    "__mycroft_skill_firstrun": false,
    "connected": true
}
```
Fixes #34

### CODE REVIEW

**Review by mikejgray — CHANGES_REQUESTED** [review]
Hi, thank you for this contribution! I appreciate it very much.

I'm not opposed to anything here, but would you please add a log message if `verbose` is True, as there will be no indication to the user if their request failed or succeeded without that. Since this is an async process that hits the PHAL plugin, which may or may not ever respond, that log would make troubleshooting issues much easier. `self.log.info()` level should be fine.

**Comment by jumper047** [comment]
Makes sense, thank you for feedback! Now it will be phrase or log entry.
BTW, I changed and force-pushed commits, is it ok or is it better to add new commits with fixes?

**Comment by mikejgray** [comment]
> Makes sense, thank you for feedback! Now it will be phrase or log entry. BTW, I changed and force-pushed commits, is it ok or is it better to add new commits with fixes?

Generally, I prefer new commits, as it makes it more clear what the history of the PR was. We squash commits on merge so there's no need to force-push anything

File: neon_homeassistant_skill/__init__.py [hunk 1]
```diff
@@ -156,7 +158,10 @@ def get_device_intent(self, message):
                     {"skill_id": self.skill_id},
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to get device status")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to get device status for {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 2]
```diff
@@ -188,7 +193,10 @@ def handle_turn_on_intent(self, message) -> None:
                     {"skill_id": self.skill_id},
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to turn on device")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to turn on device {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 3]
```diff
@@ -215,15 +224,19 @@ def handle_turn_off_intent(self, message) -> None:
                     {"skill_id": self.skill_id},
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to turn off device")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to turn off device {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 4]
```diff
@@ -294,7 +307,10 @@ def handle_set_brightness_intent(self, message: Message):
                     message.context,
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to set brightness")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to set brightness for {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 5]
```diff
@@ -330,7 +346,10 @@ def handle_increase_brightness_intent(self, message):
                     message.context,
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to increase brightness")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to increase brightness for {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 6]
```diff
@@ -348,7 +367,10 @@ def handle_decrease_brightness_intent(self, message):
                     message.context,
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to decrease brightness")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to decrease brightness for {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 7]
```diff
@@ -403,7 +425,10 @@ def handle_set_color_intent(self, message: Message):
                     message.context,
                 )
             )
-            self.speak_dialog("acknowledge")
+            if self.verbose:
+                self.speak_dialog("acknowledge")
+            else:
+                self.log.info("Trying to set light color")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to set color of {device}")
```

File: neon_homeassistant_skill/__init__.py [hunk 8]
```diff
@@ -452,7 +477,10 @@ def handle_assist_intent(self, message):
                     {"skill_id": self.skill_id},
                 )
             )
-            self.speak_dialog("assist")
+            if self.verbose:
+                self.speak_dialog("assist")
+            else:
+                self.log.info("Trying to pass message to Home Assistant's Assist API")
```
Comment (mikejgray) [inline neon_homeassistant_skill/__init__.py]:
```suggestion
                self.log.info(f"Trying to pass message to Home Assistant's Assist API:\n{command}")
```

**Review by mikejgray — CHANGES_REQUESTED** [review]

**Review by mikejgray — APPROVED** [review]

**Pull request merged**